.github/actions/nm-get-docker-tags/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Get docker image tags based on build type and the image version based on wheel name
+description: 'docker image tags for nm-vllm'
+inputs:
+  build_type:
+    description: "type of nm-vllm to install for the docker image: NIGHTLY (default) or RELEASE"
+    type: string
```

Comment:
"type" technically isn't supported for GHA action inputs. putting "type" on the inputs will cause VSCode to complain.

also i don't think we should be setting defaults in this layer. i'd prefer the caller always providing inputs. it makes things a bit clearer and avoids us having to double check every action invocation for "hidden" variables.

Reply:
Updated.